modules/ept-md-parser/client/parseFactory.js

```diff
@@ -3,13 +3,18 @@ var renderer;
 
 function parse(input) {
   if (!input) { input = ''; }
-
```

Comment:
include arbitrary newline changes in a separate commit.  review this and either specify that you intended this change, or undo it.

Reply:
These are all intended